docker-compose-conflictmonitor.yml

```diff
@@ -0,0 +1,182 @@
+services:
+  conflictmonitor:
+    profiles:
+      - conflictmonitor
+    # -------- To build the image locally --------
+    # image: jpo-conflictmonitor:latest
+    # build: ./services/intersection-api/jpo-conflictmonitor
+    #
+    # -------- To use a remote image -------------
+    image: usdotjpoode/jpo-conflictmonitor:latest
+    # --------------------------------------------
+    restart: ${RESTART_POLICY}
+    ports:
+      - '8082:8082'
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP:?error}
+      KAFKA_BOOTSTRAP_SERVERS: ${KAFKA_BOOTSTRAP_SERVERS:?error}
+    healthcheck:
+      test: ['CMD', 'java', '-version']
+      interval: 10s
+      timeout: 10s
+      retries: 20
+    logging:
+      options:
+        max-size: '10m'
+        max-file: '5'
+    deploy:
+      resources:
+        limits:
+          memory: 3G
+    depends_on:
+      kafka:
```

Comment:
There's several depends_on kafka references in this file, but I don't see kafka listed anywhere for it to depend on? Is this file not meant to be ran on its own?

Reply:
I was intending to have some dependent profiles, but I think it makes more sense to just add the dependencies to the conflictmonitor profile. That is updated now